docs/learn-staking.md

```diff
@@ -74,74 +74,72 @@ There are three different accounts for managing your funds: `Stash`, `Controller
+* [Nominator Guide](maintain-guides-how-to-nominate) - Become a nominator on the Alexander testnet.
+* [Validator Guide](maintain-guides-how-to-validate) - Become a validator on the Alexander testnet.
 
 ## Slashing
 
-Slashing will happen if a validator misbehaves(e.g. always offline, attack the network or running modified software) in the network, they and their nominators will get slashed by losing a percentage of their bonded/staked DOTs. 
+Slashing will happen if a validator misbehaves (e.g. always offline, attack the network or running modified software) in the network. They and their nominators will get slashed by losing a percentage of their bonded/staked DOTs.
 
-As validators have more DOTs staked, they will get slashed more, so we encourage nominators to shift their nominations to less popular validators to reduce the risk of being lost more. 
+As validators have more DOTs staked, they will get slashed more, so we encourage nominators to shift their nominations to less popular validators to reduce the possible losses.
 
-Based on the latest Polkadot's codebase, the following slashing have been implemented:
+Based on Polkadot's latest codebase, the following slashing conditions have been implemented:
 
 ### Unresponsiveness
 
-For every session, validators will send a "I'm Online" message to indicate they are online while unresponsiveness means that the validator fails to send the heartbeat. Depending on the repeated offences and how many other validators were offline, slashing will occur. If one-third of all validators are unresponsive, 5% of their bonded DOTs will be slashed.
+For every session, validators will send an "I'm Online" message to indicate they are online. Failure to send the heartbeat means unresponsiveness. Depending on the repeated offences and how many other validators were offline, slashing will occur. If one-third of all validators are unresponsive, 5% of their bonded DOTs will be slashed.
 
 Here is the formula for calculation:
 
     Let x = offenders, n = total no. validators
 
     Min( (3 * (x - 1)) / n, 1) * 0.05
 
-
-Validators should have a well-architected network infrastructure to ensure the node is running to reduce the risk of being slashed, for example, having high availability support to guarantee even though when the node went offline, you can still have another machine to take place.
+Validators should have a well-architected network infrastructure to ensure the node is running to reduce the risk of being slashed. A high availability setup is desirable, preferably with backup nodes that kick in **only once the original node is verifiably offline** (to avoid double-signing and being slashed for equivocation - see below), together with proxy nodes to avoid being DDoSed when your validator node's IP address is exposed. A comprehensive guide on secure validator setup is in progress with the draft available [here](https://hackmd.io/QSJlqjZpQBihEU_ojmtR8g).
```

Comment:
See Bill's comment about the published version of secure validator set-up.

Reply:
```suggestion
Validators should have a well-architected network infrastructure to ensure the node is running to reduce the risk of being slashed. A high availability setup is desirable, preferably with backup nodes that kick in **only once the original node is verifiably offline** (to avoid double-signing and being slashed for equivocation - see below), together with proxy nodes to avoid being DDoSed when your validator node's IP address is exposed. A comprehensive guide on secure validator setup is in progress with the draft available [here](https://wiki.polkadot.network/docs/en/maintain-guides-secure-validator).
```